src/vector/index.ts

```diff
@@ -130,6 +128,15 @@ async function start() {
         await settled(rageshakePromise);
 
         const fragparts = parseQsFromFragment(window.location);
+        //:tchap: get the indexedDB store version
```

Comment:
can you document why `needRefresh` check is done here, while the saving of appVersion is done later ?